alarms/alarm_manager.py

```diff
@@ -213,12 +213,11 @@ def trigger_pokemon(self, pkmn):
 
 		pkmn_info = {
 			'id': str(pkmn_id),
-			'encounter_id': pkmn['encounter_id'],
  			'pkmn': name,
 			'lat' : "{}".format(repr(lat)),
 			'lng' : "{}".format(repr(lng)),
 			'gmaps': get_gmaps_link(lat, lng),
-			'dist': get_dist_str(dist),
+			'dist': "%d%s" % (dist, 'yd' if config['UNITS'] == 'imperial' else 'm'),
```

Comment:
nice catch, no reason.  i possibly had an older version of alarm_manager.py.  reverting now.